src/ztc/api/serializers/resultaattype.py

```diff
@@ -1,106 +1,64 @@
-        view_name='api:rstiotarc-detail',
-        parent_lookup_kwargs={
-            'catalogus_pk': 'zaak_informatieobject_type__zaaktype__catalogus__pk',
-            'zaaktype_pk': 'zaak_informatieobject_type__zaaktype__pk',
-        },
-    )
-    leidtTot = NestedHyperlinkedRelatedField(
-        many=True,
+
+class ResultaatTypeSerializer(serializers.HyperlinkedModelSerializer):
+
+    zaaktype = NestedHyperlinkedRelatedField(
         read_only=True,
-        source='leidt_tot',
-        view_name='api:besluittype-detail',
+        view_name='zaaktype-detail',
+        lookup_field='uuid',
         parent_lookup_kwargs={
-            'catalogus_pk': 'catalogus__pk',
+            'catalogus_uuid': 'catalogus__uuid',
         },
+        label=_('is van')
     )
-    heeftVerplichteZaakobjecttype = NestedHyperlinkedRelatedField(
-        many=True,
-        read_only=True,
-        source='heeft_verplichte_zot',
-        view_name='api:zaakobjecttype-detail',
-        parent_lookup_kwargs={
-            'catalogus_pk': 'is_relevant_voor__catalogus__pk',
-            'zaaktype_pk': 'is_relevant_voor__pk'
-        },
+
+    _omschrijving_generiek = serializers.ReadOnlyField(
+        help_text=_("Waarde van de omschrijving-generiek referentie (attribuut `omschrijving`)")
     )
 
-    heeftVoorBrondatumArchiefprocedureRelevante = NestedHyperlinkedRelatedField(
-        read_only=True,
-        source='heeft_voor_brondatum_archiefprocedure_relevante',
-        view_name='api:eigenschap-detail',
-        parent_lookup_kwargs={
-            'catalogus_pk': 'is_van__catalogus__pk',
-            'zaaktype_pk': 'is_van__pk',
-        },
+    brondatum_archiefprocedure = BrondatumArchiefprocedureSerializer(
+        label=_("Brondatum archiefprocedure"),
+        required=False,
+        help_text=("Specificatie voor het bepalen van de brondatum voor de "
+                   "start van de Archiefactietermijn (=brondatum) van het zaakdossier.")
     )
 
     class Meta:
         model = ResultaatType
-        ref_name = model.__name__
-        source_mapping = {
-            'ingangsdatumObject': 'datum_begin_geldigheid',
-            'einddatumObject': 'datum_einde_geldigheid',
-
-            'omschrijving': 'resultaattypeomschrijving',
-            'omschrijvingGeneriek': 'resultaattypeomschrijving_generiek',
-            'brondatumProcedure': 'brondatum_archiefprocedure',
-        }
-        extra_kwargs = {
-            'url': {'view_name': 'api:resultaattype-detail'},
-        }
         fields = (
             'url',
+            'zaaktype',
             'omschrijving',
-            'omschrijvingGeneriek',
+            'omschrijving_generiek',
+            '_omschrijving_generiek',
             'selectielijstklasse',
-            'archiefnominatie',
-            'archiefactietermijn',
-            'brondatumProcedure',
+            # TODO: procestermijn + bewaartermijn + archiefnominatie
```

Comment:
Er staat TODO maar volgens mij zitten deze er allemaal in, behalve bewaartermijn.

Reply:
Klopt. Laat ik bewaartermijn nog even in TODO dan? Die hebben we niet nodig voor brondatum bepaling namelijk.

Reply:
^ ik zag dat die wel in ZRC gebruikt wordt, dus zal hem ook meteen toevoegen

Reply:
Oh goed gespot, ik had ook even gecheckt en zag em niet direct haha